packages/gafl-webapp-service/src/pages/terms-and-conditions/route.js

```diff
@@ -17,13 +17,21 @@ export const getData = async request => {
     throw new GetDataRedirect(CONTACT_SUMMARY.uri)
   }
 
-  const permission = await request.cache().helpers.transaction.getCurrentPermission()
+  const licences = transaction.permissions.map(permission => ({
+    price: permission.permit.cost
+  }))
+
   return {
-    isSalmonAndSeaTrout: permission.licenceType === mappings.LICENCE_TYPE['salmon-and-sea-trout'],
-    paymentRequired: !!Number.parseInt(permission.permit.cost)
+    paymentRequired: await priceCalculation(licences)
   }
 }
 
+const priceCalculation = async licences => {
```

Comment:
There's no reason for this to be async - so you can eliminate that and then you don't have to `await` it on line 25